ledger-api/grpc-definitions/com/daml/ledger/api/v1/commands.proto

```diff
@@ -57,20 +57,27 @@ message Commands {
   // Required
   repeated Command commands = 8;
 
+  // Specifies the deduplication period for the change ID.
+  // If omitted, the participant chooses a deduplication period at its discretion.
   oneof deduplication_period {
-      // Same semantics apply as for `deduplication_duration`.
-      google.protobuf.Duration deduplication_time = 9 [deprecated = true];
-
-      // Specifies the start of the deduplication period by a completion stream offset.
-      //
-      // Must be a valid LedgerString (as described in ``value.proto``).
-      string deduplication_offset = 15;
-
-      // Specifies the length of the deduplication period.
-      // It is interpreted relative to the local clock at some point during the submission's processing.
-      //
-      // Must be non-negative.
-      google.protobuf.Duration deduplication_duration = 16;
+    // Specifies the length of the deduplication period.
+    // Same semantics apply as for `deduplication_duration`.
+    //
+    // Must be non-negative.
+    google.protobuf.Duration deduplication_time = 9 [deprecated = true];
+
+    // Specifies the start of the deduplication period by a completion stream offset.
+    //
+    // Must be a valid LedgerString (as described in ``value.proto``).
+    string deduplication_offset = 15;
+
+    // Specifies the length of the deduplication period.
+    // It is interpreted relative to the local clock at some point during the submission's processing.
+    // If omitted, the participant will assume the configured maximum deduplication time (see
+    // ``ledger_configuration_service.proto``).
```

Comment:
Since this sits in a `oneof`, wouldn't it make more sense to move this to line 61.